mod-source-record-storage-server/src/test/java/org/folio/rest/impl/SourceStorageStreamApiTest.java

```diff
@@ -1204,6 +1204,54 @@ public void shouldReturnEmptyResponseOnSearchMarcRecordIdsWhenOffsetIs1(TestCont
     async.complete();
   }
 
+  @Test
+  public void shouldReturnIdResponseOnSearchMarcRecordIdsWhenMarcBibAndAuthoritySaved(TestContext testContext) {
+    // given
+    final Async async = testContext.async();
+    postSnapshots(testContext, snapshot_2);
+    postRecords(testContext, record_2, record_7);
```

Comment:
Will be great to rename record_2 and record_7 to another that represent type of the record. For example: auth_record_2, bib_record_1.
It will improve the readability of tests.